python/pyraphtory/tests/test_pyraphtory.py

```diff
@@ -2,4 +2,4 @@
 
 
 def test_version():
```

Comment:
We have a version file now, can be read into here?

Reply:
That is a good idea, let me try